### PR TITLE
[Chance.js] Added mac_address method

### PR DESCRIPTION
**Pull request by t1nky**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/chancejs/chancejs/blob/dc1a29a0d6963ecf4ca0bb2e69448081d621c877/chance.js

### CODE REVIEW

File: types/chance/index.d.ts [hunk 1]
```diff
@@ -92,6 +92,7 @@ declare namespace Chance {
         tld(): string;
         twitter(): string;
         url(opts?: Partial<UrlOptions>): string;
+        mac_address(opts?: Partial<MacOptions>): string;
```
Comment (peterblazejewicz) [inline types/chance/index.d.ts:95]:
```suggestion
        mac_address(opts?: MacOptions): string;
```
I think this was wrong decision to type input params as terse interfaces.
Can we switch to optional props on this interface? thx!

File: types/chance/index.d.ts [hunk 2]
```diff
@@ -224,6 +225,10 @@ declare namespace Chance {
         extensions: string[];
     }
 
+    interface MacOptions {
+        separator: string;
```
Comment (peterblazejewicz) [inline types/chance/index.d.ts:229]:
```suggestion
        separator?: string;
```
It's optional by design, never used in other shape I think
also, does it need network prop?
https://github.com/chancejs/chancejs/blob/dc1a29a0d6963ecf4ca0bb2e69448081d621c877/test/test.misc.js#L172

Comment (colbywhite) [reply types/chance/index.d.ts:229]: the use of `Partial` makes it so making each individual prop doesn't have to be marked as optional. 

but yea, adding a `networkVersion: boolean` here would be ideal.

File: types/chance/index.d.ts (hunk 1 above)
Comment (colbywhite) [reply types/chance/index.d.ts:95]: the usage of `Partial` scales better. w/o it, you would have to mark _every_ prop in most of these interfaces as optional.

**Review by colbywhite — APPROVED** [review]

**Review by colbywhite — APPROVED** [review]

**Comment by t1nky** [comment]
Ready to merge

**Pull request merged**